### DISCUSSION TITLE
转换音频cq码时发生错误

### DISCUSSION THREAD

**Discussion by GSCSDSUN** [Mrs4s/go-cqhttp#967] [Q&A]
ffmpeg装了 服务器是阿里云的轻量 系统是 Ubuntu 20
在一台server2008的机子上也会出现同样的错误

[WARNING]: 转换CQ码 [CQ:record,file=http://m10.music.126.net/20210727160439/5f6c6fbbd46b9a07f2ea4f3594e3c4b2/ymusic/13c8/9273/d588/b976bddf69daf9adf53cecb186648ae7.mp3] 时出现错误: encode silk failed: convert pcm file error: exit status 1 将原样发送.

**Comment by VnexLin** [comment]
多调用几遍就好了，重复转换可以成功（我试了第二次就发出去了）
